automation/nightly/test-nightly-build.sh

```diff
@@ -42,10 +42,14 @@ sed -i "s#docker.io/kubevirt/virt-#${kv_image/-*/-}#" deploy/images.csv
 sed -i "s#^KUBEVIRT_VERSION=.*#KUBEVIRT_VERSION=\"${kv_tag}\"#" hack/config
 sed -i "s#quay.io/kubevirt/hyperconverged-cluster-operator#${TEMP_OPERATOR_IMAGE}#" deploy/images.csv
 sed -i "s#quay.io/kubevirt/hyperconverged-cluster-webhook#${TEMP_WEBHOOK_IMAGE}#" deploy/images.csv
+eval $(source hack/config && echo ORIG_CSV_VERSION=${CSV_VERSION})
 sed -i "s#^CSV_VERSION=.*#CSV_VERSION=\"${IMAGE_TAG}\"#" hack/config
```

Comment:
Maybe we can replace the `CSV_VERSION` text in the `image.csv` file with `IMAGE_TAG`, and then we won't need to modify and restore the `hack/config` file?

Reply:
cool, done :)